backend/src/users/users.entity.ts

```diff
@@ -34,5 +34,31 @@ export class Users {
     refreshToken: string;
 
     @CreateDateColumn()
-    created_at: Date;
+    createdAt: Date;
+
+    constructor() {}
+
+    static create(
+        id: number,
+        nickname: string,
+        role: Role,
+        mainDogId: number,
```

Comment:
소유한 강아지가 없는 경우에는 mainDogId가 없어서 null type도 가능합니다.

```suggestion
        mainDogId: number | null,
```

Reply:
null을 허용 해야 하나요?